authprogs/tests/test_authprogs.py

```diff
@@ -133,6 +134,12 @@ def test_client_ip(self):
         ap = getap('0.0.0.1', 'MULTIPLE_TWO')
         self.assertRaises(authprogs.CommandRejected, ap.find_match)
 
+    def test_client_ip_cidr(self):
+        """Test that a single bare command works."""
```

Comment:
The comment needs to be updated, something like 'Test a cidr network matches'

Reply:
Good eye